src/framework/uicomponents/qml/Muse/UiComponents/sortervalue.cpp

```
@@ -19,28 +19,40 @@
+    if (role == -1) {
+        return sourceLeft < sourceRight;
+    }
 
-Qt::SortOrder SorterValue::sortOrder() const
-{
-    return m_sortOrder;
+    const QAbstractItemModel* sourceModel = proxyModel.sourceModel();
+    const QVariant leftData = sourceModel->data(sourceLeft, role);
+    const QVariant rightData = sourceModel->data(sourceRight, role);
+    const QPartialOrdering ordering = QVariant::compare(leftData, rightData);
+    if (ordering == QPartialOrdering::Unordered || ordering == QPartialOrdering::Equivalent) {
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If slots is a macro then please configure it.

(unknownMacro)